app/grandchallenge/challenges/views.py

```diff
@@ -289,3 +293,70 @@ class ChallengeViewSet(ReadOnlyModelViewSet):
     # We do not want to serialize the pk so lookup by short_name, but call it slug
     lookup_field = "short_name"
     lookup_url_kwarg = "slug"
+
+
+class OnboardingTaskList(
+    LoginRequiredMixin,
+    PermissionListMixin,
+    ListView,
+):
+    model = OnboardingTask
+    template_name = "onboardingtask_list.html"
```

Comment:
Does this template name need to be set? I thought that followed the standard naming?